lib/config/gherlintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         only_given_step_in_background: "warn",
         newline_before_scenario: "off",
         lowercase_title: "off",
-        no_but_in_given_when: "warn"
+        no_but_in_given_when: "warn",
+        require_when_and_then_step: "error",
```

Comment:
maybe we should make it `warn` by default
```suggestion
        require_when_and_then_step: "warn",
```